RELEASE.md

```diff
@@ -0,0 +1,53 @@
+# Boxo Releases
+
+## Release Policies
+### Versioning
+Boxo only releases minor and patch versions, and has no plans to change major versions from v0.
+
+Minor versions contain new or removed functionality, or significant changes in dependencies.
+
+Patch versions contain "fixes", which are generally small, targeted, and involve limited changes in functionality (such as bug fixes, security fixes, etc.). The amount of backporting depends on the severity of the issue and the impact on users...most non-critical fixes won't be backported.
+
+As a result, Boxo maintainers recommend that consumers stay up-to-date with Boxo releases.
+
+### Go Compatibility
+At any given point, the Go team supports only the latest two versions of Go released (see https://go.dev/doc/devel/release). Boxo maintainers will strive to maintain compatibilty with the older of the two supported versions, so that Boxo is also compatible with the latest two versions of Go.
+
+### Release Criteria
+Boxo does not follow a regular release cadence. Releases are performed on-demand as significant changes are made (new features added, refactorings, deprecations, etc.). The [release process](#release-process) below describes how maintainers perform a release.
```

Comment:
I know I was worried about tieing us to Kubo's release cadence, but I'm worried that this comes across as unbounded.  This was one of the fears raised by others about the repo consolidation that it would now become unpredictable when a release would get pushed out.  To alleviate fears and bring some predictability that others can plan around, I think we need to do one of two things:
1. Say we'll do a release whenever it's requested OR
2. Release at least as frequently as Kubo

Reply:
That sounds good.  Release were already unbounded on all the other repos that were moved into Boxo, they just got releases whenever anyone felt like it, so didn't seem like a big change here.